features/LoggedInCustomer.feature

```diff
@@ -0,0 +1,46 @@
+Feature: Logged In Customer
+
+Background: Logged In Customer
+    Given the user navigates to Homepage URL
+    And clicks on Sigin button
+
+Scenario: Validate Empty Logged In Customer Authentication (negative scenario)
+    Given user is in customer Sigin page 
+    When The user clicks on Sigin button
+    Then the user should see 2 error messages "This is a required field"
+
+Scenario: Emailid validation for Logged In Customer Authentication (Negative scenario)    
+    When  I fill the login email form with "<emailid>"
+    And click sigin Now button
+    Then the user able see "Please enter a valid email address"
+        Examples:
+            | emailid | 
+            | karthikeyan3071997gmail.com | 
+            | karthikeyan3071997@gmail|
+            | karthikeyan3071997.com|
+
+Scenario: Password validation for Logged In Customer Authentication (Negative scenario)    
+    When the user enters Invalid password in the required fields "<EmailID>","<password>"
+    And click a sigin button
+    Then the user should see "Incorrect CAPTCHA"
+        Examples:
+        | EmailID   | password  |
+        | karthickdk33@gmail.com  | karthick   |
+        |  karthikeyan3071997@gmail.com | 12345678   | 
+        |  karthick33@gmail.com     | karthick1234     | 
+
+Scenario: Logged In Customer Order placed (postive scenario)
```

Comment:
@karthikeyan7010 I have told already to move the order placement scenario to new feature.